packages/@aws-cdk/aws-lambda/lib/event-source-mapping.ts

```diff
@@ -35,6 +35,11 @@ export class SourceAccessConfigurationType {
    */
   public static readonly SASL_SCRAM_512_AUTH = new SourceAccessConfigurationType('SASL_SCRAM_512_AUTH');
 
+  /**
+   * The Secrets Manager ARN of your secret key containing the certificate chain (X.509 PEM), private key (PKCS#8 PEM), and private key password (optional) used for mutual TLS authentication of your MSK/Apache Kafka brokers.
```

Comment:
Last thing -- sorry I would do this myself but I don't think I can change code in your PR since you are developing on `master`... next time, if you dev on a branch, I should have default permissions to go in and change things if necessary.

What I would like to see here is to have this broken up into two lines for readability. As soon as you do it, I'll approve!

Reply:
oh thanks for the tip around branches, I'll keep that in mind 👍 new line on the comment was added in last commit